Pipfile

```diff
@@ -5,6 +5,7 @@ verify_ssl = true
 
 [packages]
 pygitguardian = { editable = true, path = "." }
+strenum = "*"
```

Comment:
Adding StrEnum here is not required: non-dev dependencies only need to be in `setup.py`.